distributed/protocol/highlevelgraph.py

```diff
@@ -127,7 +127,9 @@ def highlevelgraph_pack(hlg: HighLevelGraph, client, client_keys):
             }
         )
 
-    return msgpack.dumps({"layers": layers}, default=msgpack_encode_default)
+    return msgpack.dumps(
+        {"layers": layers}, use_bin_type=True, default=msgpack_encode_default
+    )
```

Comment:
Should we just use [`dumps_msgpack`]( https://github.com/dask/distributed/blob/7fb5d4d691df91cb08702509aeebc366bac6f5a0/distributed/protocol/core.py#L175 )?

Reply:
Yes great point. 
Initially, I didn't use `dumps_msgpack` because I was using a new ` default=` instead of `msgpack_encode_default`. Later in the development I rolled back to using `msgpack_encode_default` thus using `dumps_msgpack` now makes a lot of sense!